### PR TITLE
Add analyzers

### PR DESCRIPTION
**Pull request by crobibero**
I was also going to migrate to https://github.com/tomaspavlic/opensubtitles-api but it doesn't have a license.

### CODE REVIEW

File: Jellyfin.Plugin.OpenSubtitles/Jellyfin.Plugin.OpenSubtitles.csproj [hunk 1]
```diff
@@ -2,9 +2,13 @@
 
   <PropertyGroup>
     <TargetFramework>net5.0</TargetFramework>
-    <RootNamespace>Jellyfin.Plugin.OpenSubtitles</RootNamespace>
```
Comment (oddstr13) [inline Jellyfin.Plugin.OpenSubtitles/Jellyfin.Plugin.OpenSubtitles.csproj:5]: What is the significance of this change?

**Review by oddstr13 — APPROVED** [review]

File: Jellyfin.Plugin.OpenSubtitles/Jellyfin.Plugin.OpenSubtitles.csproj (hunk 1 above)
Comment (crobibero) [reply Jellyfin.Plugin.OpenSubtitles/Jellyfin.Plugin.OpenSubtitles.csproj:5]: This was already the root namespace so not needed

Comment (oddstr13) [reply Jellyfin.Plugin.OpenSubtitles/Jellyfin.Plugin.OpenSubtitles.csproj:5]: As specified in the `.sln`?

Comment (crobibero) [reply Jellyfin.Plugin.OpenSubtitles/Jellyfin.Plugin.OpenSubtitles.csproj:5]: It's specified by the project name + folders

**Review by h1dden-da3m0n — APPROVED** [review]
FYI, the [tomaspavlic/opensubtitles-api](https://github.com/tomaspavlic/opensubtitles-api) repo now has a License (its MIT) thanks to your issue there suggesting it.

**Pull request merged**